app/webpack.config.js

```diff
@@ -175,6 +175,9 @@ module.exports = [
         name: 'CORE_FEDERATION',
         shared: createShared(data)
       })
-    ]
+    ],
+    resolve: {
+      fallback: { util: require.resolve('util/') }
```

Comment:
Wondering why this fallback is needed. Looks like this is not needed in JupyterLab? (after quickly checking the [webpack.config.js](https://github.com/jupyterlab/jupyterlab/blob/d860b89bff1ec84309314cc686adeb1b72fc9d09/builder/src/webpack.config.base.ts#L104-L110))

Reply:
After removing this and rebuilding, I didn't reproduce the error above. Reverting this change.